.sail/Dockerfile

```diff
@@ -8,12 +8,13 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | b
 
 LABEL project_root "~/go/src/go.coder.com"
 
-# Modules break much of Go's tooling.
-ENV GO111MODULE=off
+ENV GO111MODULE=on
 
 # Install the latest version of Hugo.
 RUN wget -O /tmp/hugo.deb https://github.com/gohugoio/hugo/releases/download/v0.55.4/hugo_extended_0.55.4_Linux-64bit.deb && \
   sudo dpkg -i /tmp/hugo.deb && \
   rm -f /tmp/hugo.deb
 
+RUN mkdir -p ~/go && sudo chown -R user:user ~/go
```

Comment:
What's the purpose of the `chown`?

Reply:
Was getting a permission denied error when running `go get`.

Reply:
This should probably be documented, then.

Reply:
Just did another test with this and it does not seem to be required. I think before, I was mounting my go modules into the container instead of getting them every time. Will remove this from the PR.